lesson18.md

```diff
@@ -0,0 +1,385 @@
+<!-- .slide: id="lesson18" -->
+
+# Basic Frontend - Spring 2021
+
+Lesson 18, Tuesday, 2021-05-04
+
+---
+
+### Recap: For vs. while
+
+* We use a `for` loop if we know in advance how often it will be executed.
+
+```js
+for (let i = 0; i < 1000; i++) {
+    console.log(i);
+}
+```
+
+* We use a `while` loop if we don't know how many times the loop will be executed.
+
+```js
+while (isHungry) {
+    eat();
+}
+```
+
+---
+
+### Refresher: Data Types
+
+```js
+// string
+let name = "Owen";
+
+// number
+let age = 28;
```

Comment:
😛
```suggestion
let age = 29;
```

Reply:
maybe @haraldF  can update his script so that it computes the correct age at every semester